pyfarm/master/entrypoints.py

```diff
@@ -1,6 +1,7 @@
 # No shebang line, this module is meant to be imported
 #
 # Copyright 2013 Oliver Palmer
+# Copyright 2014, 2015 Ambient Entertainment GmbH & Co. KG
```

Comment:
My understanding of copyright notices is that, yes, you do add every year you worked on it, and in fact, the last year you worked on it is probably more important. At least, that's what I see in almost all other software copyright notices, both F/OSS and proprietary.

The idea is, if I understand things correctly, that by modifying a work, you are creating a derivative work off of the old file, which itself qualifies for copyright.